tools/topology/topology2/cavs/cavs-sdw.conf

```diff
@@ -1,18 +1,29 @@
-<include/common/vendor-token.conf>
-<include/common/tokens.conf>
-<include/common/manifest.conf>
-<include/common/pcm.conf>
-<include/common/pcm_caps.conf>
-<include/common/fe_dai.conf>
-<include/common/route.conf>
-<include/components/host.conf>
-<include/components/dai.conf>
-<include/components/pipeline.conf>
-<include/components/copier.conf>
-<include/dais/alh.conf>
-<include/dais/hw_config.conf>
-<include/pipelines/cavs/passthrough-playback.conf>
-<include/pipelines/cavs/passthrough-capture.conf>
+<searchdir:cavs>
+<searchdir:include>
+<searchdir:include/common>
+<searchdir:include/components>
+<searchdir:include/dais>
+<searchdir:include/pipelines/cavs>
```

Comment:
prefer to see include here as it makes it obvious we are including the file.

Reply:
But somehow I can't build the topology if I use include here. @ranj063 FYI

Reply:
@lgirdwoodyou can still see which conf file is included below. This will just avoid the fatigue from typing the path for every single file included below.

Reply:
> But somehow I can't build the topology if I use include here. @ranj063 FYI

Im not sure how that's possible. what error do you see @bardliao. BTW, can you add the build support for this in CMakeLists too while you're at it?

Reply:
> > But somehow I can't build the topology if I use include here. @ranj063 FYI
> 
> Im not sure how that's possible. what error do you see @bardliao. BTW, can you add the build support for this in CMakeLists too while you're at it?

I revisited the code and found that the build error is due to <cavs/common_definitions.conf> is not included in the previous cavs-sdw.conf.

Reply:
>  BTW, can you add the build support for this in CMakeLists too while you're at it?

I will try to do it.